voluncheer/voluncheer/settings.py

```diff
@@ -140,12 +140,12 @@
 
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.1/howto/static-files/
-STATIC_ROOT = os.path.join(BASE_DIR, "static")
+# STATIC_ROOT = os.path.join(BASE_DIR, "static")
 STATIC_URL = "/static/"
 # The following lines should be added while test locally.
-# STATICFILES_DIRS = [
-#     os.path.join(BASE_DIR, "static"),
-# ]
+STATICFILES_DIRS = [
```

Comment:
Does this need to be commented out for deployment on AWS? Opposite above for the `STATIC_ROOT`?

Reply:
Yes, thank you!